server_addon/maya/server/settings/publishers.py

```diff
@@ -299,6 +349,129 @@ class ExtractAlembicModel(BaseSettingsModel):
     families: list[str] = SettingsField(
         default_factory=list,
         title="Families")
+    autoSubd: bool = SettingsField(
+        title="Auto Subd",
+        description=(
+            "If this flag is present and the mesh has crease edges, crease "
+            "vertices or holes, the mesh (OPolyMesh) would now be written out "
+            "as an OSubD and crease info will be stored in the Alembic  file. "
+            "Otherwise, creases info won't be preserved in Alembic file unless"
+            " a custom Boolean attribute SubDivisionMesh has been added to "
+            "mesh node and its value is true."
+        )
+    )
+    dontSkipUnwrittenFrames: bool = SettingsField(
+        title="Dont Skip Unwritten Frames",
+        description=(
+            "When evaluating multiple translate jobs, this decides whether to "
+            "evaluate frames between jobs when there is a gap in their frame "
+            "ranges."
+        )
+    )
```

Comment:
When is this setting useful? As far as I know, the way we are exporting we are never combing a single `AbcExport` call with multiple export jobs at once - as such, this would never be used?

Reply:
gone